### PR TITLE
Documentation on model properties.

### PR DESCRIPTION
**Pull request by zahnno**
Added documentation to the following models:
- History record
- Country
- Group
- User
- Menu
- Notification
- Address

### CODE REVIEW

File: src/foam/nanos/auth/User.js [hunk 1]
```diff
@@ -289,14 +319,14 @@ foam.CLASS({
...
     {
-      class: 'DateTime',
+      class: 'Date',
       name: 'created',
-      documentation: 'Creation date'
+      documentation: 'Creation date.'
     },
     {
-      class: 'DateTime',
+      class: 'Date',
```
Comment (jlhughes) [inline src/foam/nanos/auth/User.js]: Why did you change from DateTime to Date?  These are more operational and I've been using DateTime for all CreatedAware, LastModifiedAware so when the properties are displayed we do see the time properties.

**Review by jlhughes — APPROVED** [review]
Just one question on User, but otherwise looks good.

File: src/foam/nanos/notification/Notification.js [hunk 2]
```diff
@@ -20,33 +20,33 @@ foam.CLASS({
...
       name: 'read',
+      documentation: 'Determines if notification has been read.',
       visibility: foam.u2.Visibility.RO
     },
     {
       class: 'Long',
       name: 'id',
       visibility: foam.u2.Visibility.RO
     },
-    {
```
Comment (kgrgreer) [inline src/foam/nanos/notification/Notification.js]: Why remove template property?

File: src/foam/nanos/auth/User.js (hunk 1 above)
Comment (kgrgreer) [reply src/foam/nanos/auth/User.js]: Should be DateTime.

**Comment by kgrgreer** [comment]
Thanks Patrick. Try to avoid adding code changes in documentation PR's. Knowing that a PR is only documentation or formatting changes makes it easier to review and reduces the changes of unwanted code changes slipping through. Better to split into two PR's: one containing documentation and one containing code changes (at least when the PR is mostly about documentation).

File: src/foam/nanos/notification/Notification.js (hunk 2 above)
Comment (jlhughes) [reply src/foam/nanos/notification/Notification.js]: I suspect this was a merge conflict resolution - do you remember?

**Comment by zahnno** [comment]
@kgr makes sense. I'll look out for those changes.

**Review by jlhughes — APPROVED** [review]
